lux/lib/lux/lenses/telegram/client.ex

```diff
@@ -0,0 +1,224 @@
+defmodule Lux.Lenses.Telegram.Client do
```

Comment:
Having a client is probably a good idea, but I'd implement it without rate limiting and retry by default. Mention something about this on the other PR as well.